core/block.js

```diff
@@ -741,21 +741,31 @@ Blockly.Block.prototype.getChildren = function(ordered) {
  * @param {Blockly.Block} newParent New parent block.
  */
 Blockly.Block.prototype.setParent = function(newParent) {
-  if (newParent == this.parentBlock_) {
+  if (newParent === this.parentBlock_) {
```

Comment:
Thanks for fixing this while you were in the area :D

Reply:
Happy to do so!